aries-site/src/components/content/DesignTokensTable.js

```diff
@@ -0,0 +1,170 @@
+                  id={`mode-select-${active}`}
+                  options={modeOptions}
+                  value={mode}
+                  onChange={({ option }) => {
+                    const keyPath = active.split('.');
+
+                    let res = structuredTokens;
+                    keyPath.forEach(key => {
+                      res = res[key];
+                    });
+                    handleData(getTokens(res, option));
+                    handleMode(option);
+                  }}
+                />
+              </FormField>
+            ) : undefined}
+          </Toolbar>
+          {currentData?.length > 10 ? <DataSummary /> : undefined}
+        </>
+      ) : undefined}
+      <Box
+        height={maxHeight !== false ? { max: 'large' } : undefined}
+        overflow="auto"
+      >
+        <DataTable
+          aria-describedby="token-table-heading"
+          verticalAlign="top"
+          primaryKey="token"
+          columns={[
+            {
+              property: 'id',
+              header: 'Preview',
+              render: datum => {
+                if (datum.type === 'color')
+                  return <ColorPreview datum={datum} />;
+
+                if (
+                  datum.token.includes('size') ||
+                  datum.token.includes('dimension') ||
+                  datum.token.includes('spacing') ||
+                  datum.token.includes('gap')
+                )
+                  return <DimensionPreview datum={datum} />;
+                if (
+                  datum.token.includes('radius') ||
+                  datum.token.includes('borderRadius')
+                )
+                  return <RadiusPreview datum={datum} />;
+                if (datum.token.includes('border'))
+                  return <BorderPreview datum={datum} />;
+                if (
+                  datum.token.includes('weight') ||
+                  datum.token.includes('fontWeight')
+                )
+                  return <WeightPreview datum={datum} />;
+                if (
+                  (datum.token.includes('font') &&
+                    datum.token.includes('size')) ||
+                  datum.token.includes('fontSize')
+                )
+                  return <TextPreview datum={datum} />;
+                return '--';
+              },
+              size: 'auto',
+            },
+            {
+              property: 'token',
+              header: 'Token',
+              render: datum => (
+                <Box direction="row">
+                  <Box
+                    background="background-contrast"
+                    pad="xsmall"
+                    round="xsmall"
+                    style={{ fontFamily: 'Menlo' }}
+                  >
+                    <Text size="xsmall">{datum.token}</Text>
+                  </Box>
+                  {/* <Button icon={<Copy />} size="small" /> */}
+                </Box>
+              ),
+              //   size: 'small',
```

Comment:
Can we remove these commented out lines?
```suggestion
                </Box>
              ),
```

Reply:
Removed.